deltachat-ios/Controller/NewChatViewController.swift

```diff
@@ -192,6 +192,7 @@ class NewChatViewController: UITableViewController {
                 return cell
             }
         } else if section == 1 {
```

Comment:
when touching code like that, i suggest to switch to use constants as `sectionImport`, `rowScanQr` etc (as eg. in https://github.com/deltachat/deltachat-ios/commit/74f255d6b1ede1b18d1d494e1a6ccf66cb722c60 )

if would not change the whole logic and/or use a better controller base class for now, i would just like to get rid of the numbers.

Reply:
I totally agree. In this case section 2 has not always the same meaning though. section 2 can include the "import Devices Contacts" button or the the contact list. That's why I'm not sure how rename section 2 without being super clumsy (like section 2 importOrContactListSection vs. section 3 unimportedDeviceContactListSection). Any suggestions?

Reply:
naming is hard :) i think, the point is not a super-nice-name but that the names make it possible to add sections etc. without breaking things. 

maybe use sth. has `<sectionName>Section` and then inside the section `<sectionName>Section<rowName>Row`, so eg. `unimportedSectionImportRow`. but you will know better :)